static/instant-apps-manifests/currency.yaml

```diff
@@ -0,0 +1,6 @@
+name: Currency Convertor
```

Comment:
@OrkoHunter Can we import these from the instant apps npm package somehow? It will be easier to add this yaml at the same place where we add a new app

Reply:
> @OrkoHunter Can we import these from the instant apps npm package somehow? It will be easier to add this yaml at the same place where we add a new app

That's the plan. This setup is only temporary so that we can have something up for folks to explore.